lib/stream-chat/client.rb

```diff
@@ -90,6 +90,13 @@ def set_http_client(client)
       @conn = client
     end
 
+    # Creates a JWT for a user.
+    #
+    # Stream uses JWT (JSON Web Tokens) to authenticate chat users, enabling them to login.
+    # Knowing whether a user is authorized to perform certain actions is managed
+    # separately via a role based permissions system.
+    # By default, user tokens are valid indefinitely. You can set an `exp`
```

Comment:
to be consistent with iat 
```suggestion
    # By default, user tokens are valid indefinitely. You can set an expires at (`exp`)
```

Reply:
i would prefer the other way around actually:
```
You can set an `exp` (expires at) or `iat` (issued at) claim as well.
```